src/components/Footer.jsx

```diff
@@ -9,52 +13,32 @@ const Footer = () => {
         <div className="footerin">
           <div className="w40">
             <h3>Join Our Community</h3>
-            <a href="https://www.google.com" title="Join Now">
+            <a href="https://forum.mycovidconnect.com/" title="Join Now">
               Join Now
             </a>
           </div>
           <div className="w30">
-            <ul>
-              <li>
-                <Link to="/">Contact Us</Link>
-              </li>
-              <li>
-                <Link to="/">FAQ</Link>
-              </li>
-            </ul>
-          </div>
-          <div className="w30">
-            <ul>
-              <li>
-                <Link to="/" title="About XCOV19">
-                  About XCOV19
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Work with Us">
-                  Work with Us
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Privacy Policy">
-                  Privacy Policy
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Terms &amp; Conditions">
-                  Terms &amp; Conditions
-                </Link>
-              </li>
-              <li>
-                <Link to="/" title="Press Enquiries">
-                  Press Enquiries
-                </Link>
-              </li>
-            </ul>
+            <a href="https://forum.mycovidconnect.com/d/2-about-xcov19-rapid-action-patient-dispatch-solution">
+              About XCOV19
+            </a>
+            <a href="#">Contact Us</a>
+            <div className="social-media">
```

Comment:
ask kemo for media kits, icon-logos for helpful, wprn, mattermore, jogl; opensource against covid et al, ive pasted the links to these on main whatsapp group; for the icons that kemo can provide, remove these icons and add those w/ necessary name and links